enterprise/admin/views.py

```diff
@@ -229,28 +232,47 @@ def _enroll_users(cls, enterprise_customer, emails, course_id, mode, request):
             course_id: The ID of the course in which we want to enroll
             mode: The enrollment mode the users will be enrolled in the course with
             request: The HTTP request the enrollment is being created by
+            notify: Whether to notify (by email) the users that have been enrolled
         """
         enrolled = []
         non_existing = []
         failed = []
         enrollment_client = EnrollmentApiClient()
+        course_details = CourseApiClient().get_course_details(course_id)
+        reverse_method = get_reverse_by_site(request, enterprise_customer.site)
+        course_url = reverse_method('about_course', args=(course_id,))
+        email_conn = mail.get_connection()
+        email_conn.open()
```

Comment:
You are not closing the connection - better to use it as a context manager so you don't have to explicitly close it:

```python
with mail.get_connection() as connection:
```

Also, can we avoid opening the mail connection if we are not going to be notifying the users?